pom.xml

```diff
@@ -246,7 +246,7 @@
 
   <properties>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-    <asm.version>9.0</asm.version>
+    <asm.version>9.2</asm.version>
```

Comment:
This change is analogous to #119. `Opcodes.ASM9` is still the latest non-experimental ASM API version.